grasp_utils/robot_interface/README.md

```diff
@@ -25,12 +25,6 @@ Install **robot_interface**:
 
 The installation should refer to the installation of **ros2_grasp_library**.
 
-> Note: If error "fatal error: Eigen/Geometry: No such file or directory" persists during the compilation, please check the file path of "Eigen/Geometry", if it locates at "/usr/include/eigen3/Eigen", a work around is creating a soft link to "/usr/include/Eigen" as below:
```

Comment:
I think maybe we should mention `sudo apt install ros-dashing-eigen3-cmake-module` here.

Reply:
looks good. thanks.